src/Microsoft.DotNet.Cli.Utils/IEnvironmentPath.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public interface IEnvironmentPath: IEnvironmentPathInstruction
```

Comment:
What is the need for a second interface? I have seen it being used at all.

Reply:
It is used, this interface won't give you the method requires sudo

Reply:
I see from another comment below that it will be used. Ok.